manageiq-ui-classic.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |s|
   s.executables   = s.files.grep(%r{^exe/}) { |f| File.basename(f) }
   s.require_paths = ["lib"]
 
-  s.add_dependency "rails", "~> 5.0.0", ">= 5.0.0.1"
+  s.add_dependency "rails", ">= 5.0.0.1", "< 5.1"
```

Comment:
As an FYI, my comment more is geared toward a separate PR against master (for either one of these repos) to make this consistent outside of the work you are doing here for testing Rails 5.1.0.beta1